xarray/core/variable.py

```diff
@@ -31,6 +31,7 @@
     pass
 
 
+SUPPORT_ARRAY_TYPES = (indexing.NDArrayIndexable, np.ndarray, pd.Series)
```

Comment:
For our sanity, I think this list should match the list of types accepted as valid by `as_indexable()`. So I think that would mean we should drop `pd.Series` but add `dask_array_type` (note that it's a tuple) and `pd.Index`.